core/templates/services/playthrough-issues-backend-api.service.spec.ts

```diff
@@ -107,21 +108,15 @@ describe('PlaythroughIssuesBackendApiService', () => {
 
     it('should return the playthrough data provided by the backend', fakeAsync(
       () => {
-        let backendPlaythrough = {
-          exp_id: 'exp_id1',
-          exp_version: 1,
+        let backendPlaythrough: IPlaythroughIssueBackendDict<'EarlyQuit'> = {
```

Comment:
Expose a public `IEarlyQuitPlaythroughIssueBackendDict` instead, generic parameters should not leak outside of the service.

Reply:
done